equinix/fabric_connection_schema.go

```diff
@@ -0,0 +1,840 @@
+			Elem: &schema.Resource{
+				Schema: createAccessPointInterface(),
+			},
+		},
+		"seller_region": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Access point seller region",
+		},
+		"peering_type": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.StringInSlice([]string{"PRIVATE", "MICROSOFT", "PUBLIC", "MANUAL"}, true),
+			Description:  "Peering Type",
+		},
+		"routing_protocols": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Description: "Access point routing protocols configuration",
+			Elem: &schema.Resource{
+				Schema: createFabricConnectionRoutingProtocol(),
+			},
+		},
+		"provider_connection_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Provider assigned Connection Id",
+		},
+	}
+}
+
+var createFabricConnectionSideRes = &schema.Resource{
+	Schema: createFabricConnectionSideSch(),
+}
+
+func createFabricConnectionSideSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"service_token": {
+			Type:        schema.TypeSet,
+			Optional:    true,
+			Description: "For service token based connections, Service tokens authorize users to access protected resources and services. Resource owners can distribute the tokens to trusted partners and vendors, allowing selected third parties to work directly with Equinix network assets",
+			Elem: &schema.Resource{
+				Schema: createServiceTokenSch(),
+			},
+		},
+		"access_point": {
+			Type:        schema.TypeSet,
+			Optional:    true,
+			Description: "Point of access details",
+			Elem: &schema.Resource{
+				Schema: createConnectionSideAccessPointSch(),
+			},
+		},
+		"additional_info": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Description: "Connection side additional information",
+			Elem: &schema.Resource{
+				Schema: createAdditionalInfoSch(),
+			},
+		},
+	}
+}
+
+var createRedundancyRes = &schema.Resource{
+	Schema: createRedundancySch(),
+}
+
+func createRedundancySch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"group": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Redundancy group identifier",
+		},
+		"priority": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.StringInSlice([]string{"PRIMARY", "SECONDARY"}, true),
+			Description:  "Priority type",
+		},
+	}
+}
+
+var createPortRedundancyRes = &schema.Resource{
+	Schema: createPortRedundancySch(),
+}
+
+func createPortRedundancySch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"priority": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.StringInSlice([]string{"PRIMARY", "SECONDARY"}, true),
+			Description:  "Priority type",
+		},
+	}
+}
+
+var createChangeLogRes = &schema.Resource{
+	Schema: createChangeLogSch(),
+}
+
+func createChangeLogSch() map[string]*schema.Schema {
```

Comment:
This seems like a copy of `readChangeLogSch()`. Any reason to duplicate it?

Reply:
These all are audit info attributes, customer will only see them part of, Create, Update, Delete response and is intended.